src/pages/index.js

```diff
@@ -115,36 +128,56 @@ modals.forEach((modal) => {
   });
 });
 
-// document.addEventListener("click", (evt) => {
-//   if (!evt.target.classList.contains("modal")) {
-//     closeModal(modal);
-//   }
-// });
-
 function closeModal(modal) {
   modal.classList.remove("modal_opened");
   document.removeEventListener("keyup", handleEscape);
 }
 
 function handleEditFormSubmit(evt) {
   evt.preventDefault();
-  profileName.textContent = editModalNameInput.value;
-  profileDescription.textContent = editModalDescriptionInput.value;
+  handleLoadingState(editModalSubmit, true);
+  api.editUserInfo({name:  editModalNameInput.value, about: editModalDescriptionInput.value})
+    .then((data) =>{
```

Comment:
You need to **format** the markup (and code) to show **indentation** and to make the code more readable. You can do that with `Prettier` or a combination: press `SHIFT + ALT + F`


```js
function handleEditFormSubmit(evt) {
  evt.preventDefault();
  handleLoadingState(editModalSubmit, true);
  api
    .editUserInfo({
      name: editModalNameInput.value,
      about: editModalDescriptionInput.value,
    })
    .then((data) => {
      profileName.textContent = data.name;
      profileDescription.textContent = data.about;
      closeModal(editModal);
    })
    .catch(console.error)
    .finally(() => {
      handleLoadingState(editModalSubmit, false);
    });
}

```